### DISCUSSION TITLE
[Chandler] MariaDB/Postgresql Database install

### DISCUSSION THREAD

**Discussion by mathyvds** [monicahq/monica#7151] [Q&A]
Hello

Is anyone running monica-next with a database that is not sqlite?
I have no problems installing it with a sqlite database, but installing with a datbase gives me this error:
"'unconnected' user: 'unauthenticated' host: '[secret]' (This connection closed normally without authentication)"

Can anyone possibly share their docker-compose and possible extra stept to get it running with another database?

**Comment by slifty** [comment]
Hey @mathyvds I've set up using docker compose and postgresql.

My setup is documented here: https://github.com/slifty/dsm/blob/main/monica/docker-compose.yml

Note that I'm using a manually built version of the official docker image (published to slifty/monica), since I needed FPM and v5.x which is not currently being published anywhere.  I would suggest *NOT* using my image, but you should be able to replace with whatever image you want.

If you want to share your docker-compose file I'd be glad to help debug.
